Storage/StorageTests/CoreData/MigrationTests.swift

```diff
@@ -1498,14 +1529,30 @@ private extension MigrationTests {
             "shippingPostcode": "",
             "shippingState": ""
         ])
+
+        // Required since model 75
+        if modelVersion >= 75 {
+            customer.setValue(1, forKey: "siteID")
+        }
+
+        return customer
     }
 
     /// Inserts a `CustomerSearchResult` entity, providing default values for the required properties.
     @discardableResult
-    func insertCustomerSearchResult(to context: NSManagedObjectContext) -> NSManagedObject {
-        context.insert(entityName: "CustomerSearchResult", properties: [
-            "customerID": 1
-        ])
+    func insertCustomerSearchResult(to context: NSManagedObjectContext, forModel modelVersion: Int) -> NSManagedObject {
+        if modelVersion < 75 {
+            let customerSearchResult = context.insert(entityName: "CustomerSearchResult", properties: [
+                "customerID": 1])
+            return customerSearchResult
+        } else {
+            // Required since model 75
+            let customerSearchResult = context.insert(entityName: "CustomerSearchResult", properties: [
+                "siteID": 1,
+                "keyword": ""
+            ])
+            return customerSearchResult
+        }
```

Comment:
Interestingly, here I had to use this logic rather than the one I used on `insertCustomer(to:forModel:)`, as we're removing the `customerID` attribute entirely, migration tests were failing due to incompatibility between models. Happy to refactor if there's a better way to test this.

Reply:
Makes sense. You might just want to do the old version in-line in the test which uses it, since it'll never be relevant again, but up to you on that one...

Reply:
Much more cleaner indeed. Changed here 312e9fd466f620394766d2c56c97d52b679d4480 and removed the now unused helper function here 761736be445ad848f631b91366dfe06a1e0657c8